src/locales/en.json

```diff
@@ -405,7 +405,7 @@
   "browse-page": {
     "all-no-results": "No results",
     "all-result-count": "{localeCount} result|{localeCount} results",
-    "all-result-count-more": "Over {localeCount} results",
+    "all-result-count-more": "Over {localeCount} result|Over {localeCount} results",
```

Comment:
I wonder if there's some kind of ESLint rule we could write to emit a warning or something when we find a translation string that should probably have plural variations... like maybe something basic, if the English string ends with `s` and also contains a variable interpolation? 🤔

Reply:
That's an interesting idea, @sarayourfriend !